docs/documentation/cert-manager.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Cert-Manager
+description: Cert-Manager
+layout: docs
+permalink: /docs/cert-manager
+---
+### Cert-Manager
+
+Dekorate supports generating `Certificate` and `Issuer` resources using [Cert-Manager](https://cert-manager.io/). When these resources are loaded into the cluster, Cert-Manager will issue the certificates, issuers, and trust-store files that then can be used by final applications.
+
+To let Dekorate to generate the certificate and issuer resources, simply declare the following dependency part of your pom file:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>certmanager-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+And provide the certificate configuration. The minimal information that the Dekorate needs are:
+- `secretName` : the name of the Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) resource that will contain all the generated files by Cert-Manager.
+- the Issuer that represents certificate authorities (CAs). See all the supported options in [the Issuer](#issuers) section.
+
+For all the configuration options, please go to [the Configuration guide](https://dekorate.io/configuration-guide/#cert-manager) for Cert-Manager.
+
+To provide the minimal configuration, we can do it via the properties file:
```

Comment:
"To provide the minimal configuration, we can do it via the properties file:" --> "The minimal configuration can be provided using the properties file and the following keys:"